packages/gap-web-ui/src/components/question-page/inputs/Checkboxes.tsx

```diff
@@ -73,6 +74,86 @@ const Checkboxes = ({
+
+              const getDataBehaviour = () => {
+                if (!divideLastCheckboxOption) return undefined;
+                if (divideCheckboxIndex === 1 && index === 0)
+                  return 'exclusive';
+                if (
+                  divideCheckboxIndex === disabledOptions.length - 1 &&
+                  index === disabledOptions.length - 1
+                )
+                  return 'exclusive';
+                return undefined;
+              };
+
+              return (
+                <React.Fragment key={value}>
+                  {divideLastCheckboxOption &&
+                    index === divideCheckboxIndex && (
+                      <div className="govuk-checkboxes__divider">or</div>
+                    )}
+
+                  <div className="govuk-checkboxes__item">
+                    <input
+                      className="govuk-checkboxes__input"
+                      id={index === 0 ? fieldName : `${fieldName}-${index + 1}`}
+                      name={fieldName}
+                      type="checkbox"
+                      value={value}
+                      defaultChecked={
+                        defaultCheckboxes &&
+                        defaultCheckboxes.indexOf(value) >= 0
+                      }
+                      disabled={true}
+                      data-behaviour={getDataBehaviour()}
+                      data-cy={`cy-checkbox-value-${value}`}
+                      data-aria-controls={
+                        typeof option === 'object' && option.conditionalInput
+                          ? `conditional-${
+                              index === 0
+                                ? fieldName
+                                : `${fieldName}-${index + 1}`
+                            }`
+                          : undefined
+                      }
+                    />
+                    <label
+                      className="govuk-label govuk-checkboxes__label"
+                      htmlFor={
+                        index === 0 ? fieldName : `${fieldName}-${index + 1}`
+                      }
+                      data-cy={`cy-checkbox-label-${value}`}
+                    >
+                      {typeof option === 'string' ? value : option.label}
+                    </label>
+                  </div>
+
+                  {typeof option === 'object' && option.conditionalInput && (
+                    <div
+                      className="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden"
+                      id={`conditional-${
+                        index === 0 ? fieldName : `${fieldName}-${index + 1}`
+                      }`}
+                      data-testid={`conditional-input-wrapper-${index + 1}`}
+                    >
+                      <div className="govuk-form-group">
+                        {option.conditionalInput}
+                      </div>
+                    </div>
+                  )}
```

Comment:
not seeing where this is actually used either

Reply:
Yeah was definitely reluctant to go with this approach due to the duplication, have made it messy and added some unnecessary snippets without fully understanding the UI components 😅

Will have a go at it after lunch and try to make the disabling cleaner based on your suggestion above. Thanks for the feedback on this! Will reach out if I get lost